SPECS-EXTENDED/whois/whois.spec

```
@@ -1,26 +1,37 @@
+Summary:        Improved WHOIS client
+Name:           whois
+Version:        5.5.15
+Release:        1%{?dist}
+License:        GPLv2+
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
+URL:            https://www.linux.it/~md/software/
+Source0:        http://ftp.debian.org/debian/pool/main/w/%{name}/%{name}_%{version}.tar.xz
+%global genname whois
+%global alternative md
+%global cfgfile %{name}.conf
```

Thanks for linting the spec. Just a few updates, because the linter is still not fully configured:
```suggestion
%global alternative md
%global cfgfile %{name}.conf
%global genname whois

Summary:        Improved WHOIS client
Name:           whois
Version:        5.5.15
Release:        1%{?dist}
License:        GPLv2+
Vendor:         Microsoft Corporation
Distribution:   Mariner
URL:            https://www.linux.it/~md/software/
Source0:        http://ftp.debian.org/debian/pool/main/w/%{name}/%{name}_%{version}.tar.xz
```